src/shared/containers/AppContainer.js

```diff
@@ -1,13 +1,20 @@
 import { h } from 'preact';
+import { connect } from 'preact-redux';
+import { createStructuredSelector } from 'reselect';
+
 import { Header } from './HeaderContainer';
 import { Layout } from '../../components/Layout';
 
 
-const AppContainer = ({ children }) => (
+const PureAppContainer = ({ children, ready }) => (
     <Layout>
         <Header />
-        { children }
+        { ready ? children : null }
     </Layout>
 );
 
+const AppContainer = connect(createStructuredSelector({
+    ready: state => state.username !== '',
```

Comment:
Potentially `''` is a valid user name. FES sends either what it got from auth, or `undefined`